modules/Bio/EnsEMBL/Exon.pm

```diff
@@ -1281,8 +1281,7 @@ sub is_constitutive {
 
 =head2 is_coding
 
-  Arg [1]    : Boolean $is_coding
-  Arg [2]    : Bio::EnsEMBL::Transcript
+  Arg [1]    : Bio::EnsEMBL::Transcript
   Example    : $exon->is_coding()
   Description: Says if the exon is within the translation or not
   Returntype : Int
```

Comment:
I know this isn't part of your changes, but shouldn't the return type be a Boolean or something like that?

Reply:
We should, if perl had a boolean type :)